test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.IsInstanceOfTypeTests.cs

```diff
@@ -20,53 +20,90 @@ namespace Microsoft.VisualStudio.TestPlatform.TestFramework.UnitTests;
 public partial class AssertTests
 {
     [TestMethod]
-    public void InstanceOfTypeShouldFailWhenValueIsNull()
+    public void IsInstanceOfType_WhenValueIsNull_Fails()
     {
         static void action() => TestFrameworkV2.Assert.IsInstanceOfType(null, typeof(AssertTests));
         ActionUtility.ActionShouldThrowExceptionOfType(action, typeof(TestFrameworkV2.AssertFailedException));
     }
 
     [TestMethod]
-    public void InstanceOfTypeShouldFailWhenTypeIsNull()
+    public void IsInstanceOfType_WhenTypeIsNull_Fails()
     {
         static void action() => TestFrameworkV2.Assert.IsInstanceOfType(5, null);
         ActionUtility.ActionShouldThrowExceptionOfType(action, typeof(TestFrameworkV2.AssertFailedException));
     }
 
     [TestMethod]
-    public void InstanceOfTypeShouldPassOnSameInstance()
+    public void IsInstanceOfType_OnSameInstance_Passes()
```

Comment:
Maybe you could rename `Passes` with `DoesNotThrow` or something similar so that when coming back to these we don't feel like we forgot to test something. (Applies to all tests of this class).